addons/l10n_be/models/template_be.py

```diff
@@ -80,9 +86,12 @@ def _get_be_reconcile_model(self):
                         'label': 'Frais bancaires TVA21',
                     }),
                 ],
+                'name@fr': 'Frais bancaires TVA21',
+                'name@nl': 'Bankkosten exclusief 21% btw',
+                'name@de': 'Bankgebühren inkl. MwSt. 21 %',
             },
             'virements_internes_template': {
-                'name': 'Virements internes',
+                'name': 'Internal transfers',
```

Comment:
having it in english doesn't seem too, maybe you could just use french as default

Reply:
What do you mean by "-u" and "-u account"?

>having it in English doesn't seem too, maybe you could just use french as default
>

I talked to TSB and he suggested using it in English as default and add translations for other languages

Reply:
@nle-odoo You can also go to Account -> Configuration -> settings -> Click Reload inside Fiscal Localization and it will update the chart template